docs/src/main/sphinx/functions/binary.rst

```diff
@@ -172,7 +172,11 @@ Hashing functions
 
 .. function:: murmur3(binary) -> varbinary
 
-    Computes the 128-bit murmur3 hash of ``binary``.
```

Comment:
Let's keep the text the same, so that it's consistent with the others, but add a link:
```
Computes the 128-bit `MurmurHash3 <https://en.wikipedia.org/wiki/MurmurHash>`_ hash of ``binary``.

Reply:
Ok, done.